### ISSUE TITLE
Map: "Go to here" - does not engange Guided mode

### ISSUE THREAD

**Issue by AndKe** [ArduPilot/apm_planner#993]
Go to here, or go to here (alt)  - right click in map does not switch to guided mode.

**Comment by billbonney** [comment]
The workaround is to select Guided Mode and then select the options.

**Comment by AndKe** [comment]
Workaround is clear, but if we see this feature as a safety feature, it
needs to do it automatically to save time. Do you agree ?

**Comment by billbonney** [comment]
I don't disagree. Just highlighting the workaround.

**Comment by AndKe** [comment]
maybe this issue should remain open ?

**Comment by Arne-W** [comment]
Yes, should remain open! And marked to be related to #1157 :smiley:
